vhdbuilder/packer/install-dependencies.sh

```diff
@@ -130,6 +130,9 @@ if ! isMarinerOrAzureLinux "$OS"; then
 fi
 capture_benchmark "${SCRIPT_NAME}_validate_container_runtime_and_override_ubuntu_net_config"
 
+# Configure SSH service during VHD build for Ubuntu 22.10+
+configureSSHService "$OS" "$OS_VERSION" || echo "##vso[task.logissue type=warning]SSH Service configuration failed, but continuing VHD build"
+
```

Comment:
I think we can also add `capture_benchmark` to capture the timing for future use. If it's very short, that's idea. If one day we find it unusually long, that means there must be a bug somewhere related.